unittest.sh

```diff
@@ -61,13 +68,17 @@ do
         continue
     fi
 
+    # Ignore script folder when in qmlui mode
+    if [ "${test}" == "engine/test/script" -a "$TARGET" == "qmlui" ]; then
```

Comment:
If I understand correctly. this is a workaround because the current Script test unit doesn't work with ScriptV4 class?
If so, we should fix the test, not the sh script ;)

Reply:
Good catch. Thanks for your review, I was chewing on getting the build and the tests for v5 running on Linux for quite a while (on a very low priority) and had forgotten about this deactivation.
Please see https://github.com/mcallegari/qlcplus/pull/1283 as a basis for the fix which (when integrated) I will merge here together with the fix of above lines.

Reply:
This can be removed now and also in unittest.bat.
Also, this can be changed too:
https://github.com/mcallegari/qlcplus/blob/master/engine/test/test.pro#L42